nearcore/src/config.rs

```diff
@@ -334,6 +334,33 @@ fn default_enable_rocksdb_statistics() -> bool {
     false
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct StoreConfig {
+    /// Maximum number of store files being opened simultaneously.
+    #[serde(default = "default_max_open_files")]
+    pub max_open_files: i32,
+    /// Cache size for ColState column.
+    #[serde(default = "default_col_state_cache_size")]
+    pub col_state_cache_size: usize,
+}
+
+impl Default for StoreConfig {
+    fn default() -> Self {
+        StoreConfig {
+            max_open_files: default_max_open_files(),
+            col_state_cache_size: default_col_state_cache_size(),
+        }
+    }
+}
+
+fn default_max_open_files() -> i32 {
+    10 * 1000
+}
+
+fn default_col_state_cache_size() -> usize {
+    512 * 1024 * 1024
```

Comment:
mention where did we get this number (I guess here, we can say, that it was set like this for a long time ;-) and we might consider tuning this number in the future.